src/Sarif/Core/Rule.cs

```diff
@@ -13,5 +13,10 @@ public string Format(string messageId, IEnumerable<string> arguments)
         {
             return string.Format(CultureInfo.CurrentCulture, this.MessageStrings[messageId], arguments.ToArray());
         }
+
+        public bool ShouldSerializeDeprecatedIds()
+        {
+            return this.DeprecatedIds?.Count(e => e != null) > 1;
```

Comment:
>1 [](start = 63, length = 1)

Shouldn't this be `> 0` ?

Reply:
Oops. All changes introduce change of regression, even small ones. :)

---
In reply to: [249228972](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249228972) [](ancestors = 249228972)